c7n/schema.py

```diff
@@ -28,12 +28,16 @@
 from collections import Counter
 import json
 import logging
+import yaml
+import inspect
 
 from jsonschema import Draft4Validator as Validator
 from jsonschema.exceptions import best_match
 
 from c7n.manager import resources
+from c7n.resources import load_resources
 from c7n.filters import ValueFilter, EventFilter, AgeFilter
+from c7n.utils import ArgumentError
```

Comment:
let's stick exceptions into an exceptions.py module we've got some others i'd like to migrate to a better location then importing around the code base from different locations.